annotations/src/main/java/org/robolectric/versioning/AndroidVersions.java

```diff
@@ -89,17 +90,10 @@ public abstract static class AndroidRelease implements Comparable<AndroidRelease
      *
      * @param other the object to be compared.
      * @return 1 if this is greater than other, 0 if equal, -1 if less
-     * @throws IllegalStateException if other is not an instance of AndroidRelease.
+     * @throws NullPointerException if other is null.
```

Comment:
This is actually not a chance in behavior.
In the removed `if` below, `other` was used after we checked that it is `null`. So a `NullPointerException` is already thrown.